assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_student_list_submissions.vm

```diff
@@ -138,8 +138,10 @@
 						#else
 							#set($assignments=false)
 							#set($assignments=$!studentAssignmentsTable.get($member))
-							#foreach ($assignment in $!assignments)
-								#if ($assignment.isGroup)
+							#foreach ($assignment in $assignments)
+								#set($groupIds='')
+								#set($groupIds=$assignment.Groups)
+								#if ($assignment.IsGroup || ($groupIds != '' && $groupIds.size() > 0))
```

Comment:
```suggestion
								#if ($assignment.TypeOfAccess.Ordinal == 1)
```
This if covers when groups are in use i.e. either the assignment is group based or it is released to groups

Reply:
@ern thank you for your comment. Changed!